### PR TITLE
[CPDLP-3518] Add migration queue only worker to NPQ

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-3518

### Changes proposed in this pull request

* Migration jobs will use `migration` queue
* Existing worker runs the following queues: `default,low_priority,participant_outcomes`
* New migration worker runs `migration` queue only

### CODE REVIEW

File: terraform/application/application.tf [hunk 1]
```diff
@@ -69,11 +69,36 @@ module "worker_application" {
 
   docker_image = var.docker_image
 
-  command       = ["bundle", "exec", "rake", "jobs:work"]
+  command       = ["/bin/sh", "-c", "QUEUES=default,low_priority,participant_outcomes bundle exec rake jobs:work"]
```
Comment (ethax-ross) [inline terraform/application/application.tf:72]: Worth adding a comment in the delayed_job initializer pointing to this? I'm just thinking if someone adds a queue in the future they would expect it to be picked up by default and not realise they'll need to add it here too

**Review by ethax-ross — COMMENTED** [review]
Looks good, has this been deployed to migration yet?

File: app/jobs/parity_check_job.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class ParityCheckJob < ApplicationJob
-  queue_as :high_priority
+  queue_as :migration
```
Comment (cwrw) [inline app/jobs/parity_check_job.rb:2]: I don't think this needs to move to migration 🤔

**Review by cwrw — COMMENTED** [review]
looking good 🙌  happy to approve once manually we run plan/apply on migration and all is well :)

File: app/jobs/parity_check_job.rb (hunk 2 above)
Comment (mooktakim) [reply app/jobs/parity_check_job.rb]: changed it back

**Review by leandroalemao — APPROVED** [review]
Approving so it can be tested in migration 👍  tks @mooktakim 👍

**Pull request merged**